includes/admin/class-sensei-learners-main.php

```diff
@@ -15,44 +21,104 @@
  */
 class Sensei_Learners_Main extends Sensei_List_Table {
 
-	public $course_id = 0;
-	public $lesson_id = 0;
-	public $view      = 'courses';
-	public $page_slug = 'sensei_learners';
+	/**
+	 * The course id of the current view.
+	 *
+	 * @var integer
+	 */
+	private $course_id;
```

Comment:
I'm fine with it, but just wanted to note in the PR that these were public and are now private. I don't think any of our plugins touch this, but just in case others run into issues.